src/poprox_recommender/test_json.py

```diff
@@ -86,12 +87,29 @@ def recsys_metric(recommendations, row_index, news_struuid_ID ):
     account_id = list(recommendations.keys())[0]
     recommended_list = recommendations[account_id]
     recommended_list = [news_struuid_ID[item.url] for item in recommended_list]
-
-    single_mrr = compute_mrr(recommended_list, impressions_truth)
-    single_ndcg5 = compute_ndcg(recommended_list, impressions_truth, 5)
-    single_ndcg10 = compute_ndcg(recommended_list, impressions_truth, 10)
-
-    return single_ndcg5, single_ndcg10, single_mrr
+    
+    
+    recs = pd.DataFrame({
+        'item': recommended_list
+    })
+    
+    truth = pd.DataFrame.from_records(
+        (
+            (row.split('-')[0], int(row.split('-')[1]))
+            for row in impressions_truth
+        ),
+        columns=['item', 'rating']
+    ).set_index('item')
+
+    single_rr = topn.recip_rank(recs, truth)
+    single_ndcg5 = topn.ndcg(recs, truth, k=5)
+    single_ndcg10 = topn.ndcg(recs, truth, k=10)
+
+    # single_rr = compute_mrr(recommended_list, impressions_truth)
+    # single_ndcg5 = compute_ndcg(recommended_list, impressions_truth, 5)
+    # single_ndcg10 = compute_ndcg(recommended_list, impressions_truth, 10)
```

Comment:
We have this code in the commit history, so it's safe to remove instead of leaving it commented out because we can still retrieve it later if we need it:
```suggestion
    # single_rr = compute_mrr(recommended_list, impressions_truth)
    # single_ndcg5 = compute_ndcg(recommended_list, impressions_truth, 5)
    # single_ndcg10 = compute_ndcg(recommended_list, impressions_truth, 10)
```